contrib/resources/translations/it.lng

```diff
@@ -3139,18 +3136,18 @@ Comando mancante per il canale [color=light-cyan]%s[reset]
 Comando non valido per il canale [color=light-cyan]%s[reset]: [color=white]%s[reset]
 .
 :PROGRAM_MODE_HELP_LONG
-Imposta la modalità di visualizzazione o la frequenza tipomatica della tastiera.
+Imposta la modalità di visualizzazione o la frequenza tipomatica.
 
 Utilizzo:
   [color=light-green]mode[reset] [color=white]COLONNExRIGHE[reset]
   [color=light-green]mode[reset] [color=white]COLONNE,RIGHE[reset]
-  [color=light-green]mode[reset] rate=[color=white]FREQ[reset] delay=[color=white]RITARDO[reset]
+  [color=light-green]mode[reset] rate=[color=white]FREQUENZA[reset] delay=[color=white]RITARDO[reset]
 
 Parametri:
   [color=white]COLONNE[reset]   numero di caratteri (colonne) per riga (40, 80, or 132)
   [color=white]RIGHE[reset]     numero di righe sullo schermo (25, 28, 30, 34, 43, 50, o 60)
-  [color=white]FREQ[reset]      freq. ripetizione caratteri, da [color=white]1[reset] (lento) a [color=white]32[reset] (veloce)
-  [color=white]RITARDO[reset]   rit. ripetizione caratteri, da [color=white]1[reset] (minore) a [color=white]4[reset] (maggiore)
+  [color=white]FREQUENZA[reset] frequenza di ripetizione caratteri, da [color=white]1[reset] (lento) a [color=white]32[reset] (veloce)
```

Comment:
Il motivo per cui queste due righe sono leggermente diverse dall'originale è per farle entrare in 80 caratteri.